### PR TITLE
added dark mode

### PR DESCRIPTION
**Pull request by Abhinav-3359**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,258 +1,469 @@
...
+      rel="stylesheet"
+    />
+    <!-- font awesome cdn -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <!-- Favicon -->
+    <link rel="icon" type="image/x-icon" href="./assets/images/favicon.png" />
+    <style>
```
Comment (ShehrozIrfan) [inline index.html:31]: hey @Abhinav-3359 !
Add this internal css in the`assets/css/style.css` file.

File: index.html [hunk 2]
```diff
@@ -1,258 +1,469 @@
...
+        background-color: white;
+        color: black;
+      }
+    </style>
+  </head>
+  <body>
+    <h3 id="DarkModetext">Dark Mode is OFF</h3>
+    <button onclick="darkMode()">Darkmode</button>
+    <button onclick="lightMode()">LightMode</button>
+    <script>
```
Comment (ShehrozIrfan) [inline index.html:52]: Hey @Abhinav-3359 !
Add this javascript in `assets/javascript/app.js` file and include it in the `index.html` file.

File: index.html [hunk 3]
```diff
@@ -1,258 +1,469 @@
...
+                    ></i
+                  ></a>
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
```
Comment (ShehrozIrfan) [inline index.html:467]: remove this extra line.

File: index.html (hunk 1 above)
Comment (Abhinav-3359) [reply index.html]: sir pls review

File: style.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  font-size: 15px;
+}
+
+@media(max-width: 300px) {
+  .profile-image {
+    width: 150px;
+    height: 150px;
+  }
+}
+body {
```
Comment (ShehrozIrfan) [inline style.css:59]: Add the `body` styles on line `6`

File: style.css [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+  padding: 25px;
+  background-color: white;
+  color: black;
+  font-size: 25px;
+}
+.dark-mode {
```
Comment (ShehrozIrfan) [inline style.css:65]: Add all your styles just above the `@media(max-width: 300px) `

File: style.css [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+  background-color: black;
+  color: white;
+}
```
Comment (ShehrozIrfan) [inline style.css:68]: Make sure to line space after each class, as already in the file for each style.

**Comment by ShehrozIrfan** [comment]
@Abhinav-3359 
It seems that JavaScript code is missing. Would you please verify?

**Comment by ShehrozIrfan** [comment]
@Abhinav-3359 
There are conflicts in your PR. Remove them. 
Also I don't see any JavaScript code. Make sure you've added that, and dark and light mode are working alright on your localhost

**Comment by ShehrozIrfan** [comment]
@Abhinav-3359 
Is it possible for you to have a look on this issue: #30

**Comment by Abhinav-3359** [comment]
Okay sir I’ll try

**Comment by ShehrozIrfan** [comment]
@Abhinav-3359 
Send a message on the issue: #30 
So, that I can assign this to you!

**Pull request closed without merging**